segmentation_models_pytorch/__version__.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (0, 4, 0)
+VERSION = (0, 4, "1dev0")
```

Comment:
According to [PEP 440](https://peps.python.org/pep-0440/), it should be:
```suggestion
VERSION = (0, 4, 1, "dev0")
```

Reply:
Thanks, didn't know that :)